keyboards/gmmk/pro/ansi/keymaps/stickandgum/config.h

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2021 stickandgum
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// I like NKRO.. force it! - You can still toggle it if you want to disable.
+#define FORCE_NKRO

```

Comment:
Honestly, I'm not a fan of having it enabled like this.  Some BIOSes/UEFI firmware have issues with NKRO. 

It should be noted, though, that once it's enabled, the setting is stored in eeprom and is retained. 

You don't need to change it, but more of a "heads up".